src/Lykke.Job.Messages.Contract/Lykke.Job.Messages.Contract/Lykke.Job.Messages.Contract.csproj

```diff
@@ -4,10 +4,6 @@
     <Version>1.0.1</Version>
   </PropertyGroup>
 
-  <ItemGroup>
```

Comment:
is it ok ?

Reply:
it works without this section.
Seems ok.